README.md

```diff
@@ -99,6 +99,10 @@ return static function (Config\RectorConfig $rectorConfig): void {
 
 Faker is released under the MIT License. See [`LICENSE`](LICENSE) for details.
 
+## PHP Version Support Policy
+
+The maintainers of this package add support for a PHP version following its initial release and drop support for a PHP version one year after it has reached its end of security support.
```

Comment:
This may or may not reflect the current implicit PHP version support policy.

Reply:
I'd rather make this 2 or 3 years.

Reply:
I honestly think one full year after *security support* ends, is already pretty long.

Reply:
This is only free security support from php.net. Many users receive extended support from third parties, both free and paid. Given faker is a library used for testing, it is convenient to support as wide a range of PHP versions as possible.

Reply:
@GrahamCampbell 

> I'd rather make this 2 or 3 years.

If we drop support for a PHP version one year after it has reached its end of security support, we would need to support

- `7.4`
- `8.0`
- `8.1`
- `8.2`

today (which we currently do).

If we drop support for a PHP version two years after it has reached its end of security support, we would need to support

- `7.3`
- `7.4`
- `8.0`
- `8.1`
- `8.2`

today.

If we drop support for a PHP version three years after it has reached its end of security support, we would need to support

- `7.2`
- `7.3`
- `7.4`
- `8.0`
- `8.1`
- `8.2`

today.

Reply:
People rarely complain, but they do suffer. 😆

Reply:
I thought we decided to drop 7.x on the 2.x branches anyways?

But lets have a call on it and go forward. Left or right all directions are ok and there are always pros and cons for both directions.

Being stuck and nothing makes it even worse

Reply:
we could just have no formal policy on php version support. ;)

Reply:
The policy could also be:

- support for PHP versions will be offered until there's justifiable reason to drop support for an older PHP version.
- Support for a PHP version can only be dropped at least 1 year after it's security release has ended, provided justifiable reason has been provided.

Reply:
@deleugpn 

What would be a justifiable reason?

Reply:
@GrahamCampbell 

> we could just have no formal policy on php version support. ;)

Sure, we could, but ...

> and do the sensible thing on applicable branches

...we could also repeat discussing this every other week, month, or year until we lose interest contributing and eventually abandon this library - but: I'm not getting paid for this, and neither are any of you. 

The sensible thing to do for all of us is to avoid wasting resources.

A clear, actionable PHP version support policy can help us with that.

Reply:
@GrahamCampbell 

I understand and appreciate that we can all change our minds, but see https://github.com/FakerPHP/Faker/pull/3#issuecomment-718147651:

> I am personally not happy with supporting super old PHP versions. I think 7.1 makes for a good lower-bound in 2020. People using deprecated PHP can still use the 1.9.1 release. My view is that if you are happy to use super old PHP, then using an old version of a library comes with the territory.